apps/nuxt3-ssr/interfaces/Icohort.ts

```diff
@@ -15,9 +15,11 @@ interface ICohort {
   endYear: number;
   countries: {
     name: string;
```

Comment:
You could consider to make this and IOntology type? So you can reuse it?

Reply:
Yes i need to cleanup the interface list , and use some more general types . However is really nice to remove all unused props from the gql request ( and types ) to safe some bytes on each request. The end result is maybe that the gql should define the types instead of keeping stuff in sync